### ISSUE TITLE
Reduce External deps (e.g ngx-bootstrap)

### ISSUE THREAD

**Issue by abbasogaji** [webcat12345/ngx-intl-tel-input#378]
it would be nice to have an out-of-the-box feel with this library. where deps are not external, (i.e you dont need to add mulitple dependencies before it works e.t.c)

for example;  example in my project i use ngboostrap already, but trying to install this library i will need to install ngx-bootstap also

**Comment by MixTerritory** [comment]
I'm agree with this request. I don't use ngx-bootstrap in my project. For my project it looks like an unoptimized approach, to install  additionally ngx-bootstrap (>8Mb) when I want to use just ngx-intl-tel-input (~130Kb)

**Comment by cmckni3** [comment]
I agree with this. Shouldn't depend on CSS frameworks. Currently, only the dropdown component from ngx-bootstrap is used.

**Comment by VictorDev200** [comment]
Hello, Is there any help that I can do on this? I'm having issues with Angular v20 and the ngx-bootstrap dependency

**Comment by cmckni3** [comment]
> Hello, Is there any help that I can do on this? I'm having issues with Angular v20 and the ngx-bootstrap dependency

@VictorDev200, sure, a PR to rework the dropdown would be welcome.

I plan to finish up the Angular 18 PR when I have a chance.
